tests/mtest/src/main/scala/tests/GeneralPCSuite.scala

```diff
@@ -0,0 +1,100 @@
+package tests
+
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Path
+
+import scala.util.control.NonFatal
+
+import scala.meta.Dialect
+import scala.meta.dialects
+import scala.meta.internal.jdk.CollectionConverters._
+import scala.meta.internal.metals.ClasspathSearch
+import scala.meta.internal.metals.Docstrings
+import scala.meta.internal.metals.ExcludedPackagesHandler
+import scala.meta.internal.metals.JdkSources
+import scala.meta.io.AbsolutePath
+
+import coursierapi.Fetch
+import coursierapi.Repository
+import munit.Assertions.fail
+
+trait GeneralPCSuite {
```

Comment:
```suggestion
trait PCSuite {
```
Would probably be better or `PresentationCompilerSuite`